test/android/app/src/main/assets/www/index.html

```diff
@@ -0,0 +1,56 @@
+<!--
```

Comment:
Should this be in the `test/androidx` directory instead of the `test/android` folder ?

Reply:
I deleted this file. It is not needed.